packages/fxa-settings/src/pages/Signin/SigninRecoveryPhone/index.tsx

```diff
@@ -0,0 +1,179 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import AppLayout from '../../../components/AppLayout';
+import FormVerifyTotp from '../../../components/FormVerifyTotp';
+import { RouteComponentProps } from '@reach/router';
+import { useFtlMsgResolver } from '../../../models';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { BackupRecoveryPhoneCodeImage } from '../../../components/images';
+import Banner from '../../../components/Banner';
+import { HeadingPrimary } from '../../../components/HeadingPrimary';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import ButtonBack from '../../../components/ButtonBack';
+import { getLocalizedErrorMessage } from '../../../lib/error-utils';
+import { AuthUiErrors } from '../../../lib/auth-errors/auth-errors';
+import { SigninRecoveryPhoneProps } from './interfaces';
+
+const SigninRecoveryPhone = ({
+  lastFourPhoneDigits,
+  verifyCode,
+  resendCode,
+}: SigninRecoveryPhoneProps & RouteComponentProps) => {
+  const [errorMessage, setErrorMessage] = React.useState('');
+  const [errorDescription, setErrorDescription] = React.useState('');
+  const [showResendSuccessBanner, setShowResendSuccessBanner] =
+    React.useState(false);
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const maskedPhoneNumber = `+1-•••-${lastFourPhoneDigits}`;
```

Comment:
You may want to update this to just the dots, for consistency across add/delete inside Settings. In that new ticket I referenced, I think we can iron this out with that `national_format` from Twilio if we want these formatted properly.